SPECS/fmt/fmt.spec

```diff
@@ -1,6 +1,6 @@
 Summary:        Small, safe and fast formatting library for C++
 Name:           fmt
-Version:        8.1.1
+Version:        10.1.0
```

Comment:
Curious if we can move to 10.1.1 which is the current release

Reply:
went to `10.2.1` which was more recent.